src/index.html

```diff
@@ -9,10 +9,18 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/main.scss"
```

Comment:
You are linking a SCSS file in the HTML. The browser does not understand SCSS. You should compile your SCSS to CSS and link the CSS file here instead.